packages/shared/src/graphql/fragments.ts

```diff
@@ -215,6 +215,7 @@ export const FEED_POST_INFO_FRAGMENT = gql`
       }
     }
     slug
+    clickbaitTitleDetected
```

Comment:
Was going to comment not sure on naming as it's just any title eventually that we find optimizing for.
But since we called the feature clickbait shield, let's roll